### ISSUE TITLE
Adapt statstics for journal_issue

### ISSUE THREAD

**Issue by zuphilip** [locdb/locdb-frend#423]
The `JOURNAL_ISSUE` has never any contributors, so we should not list `missingContributors` for that type. CC @anlausch @kleinann

**Comment by zuphilip** [comment]
Or should the publisher be saved for each journal issue instead of saving the publisher for the journal article?

**Comment by kleinann** [comment]
Yes, I think the publisher belongs to the journal issue, in principle (and publishing year, too). But it's true that in our current conversion of the Crossref data, both are only saved in the journal article data. I'm not so sure if they should be saved only in the issue instead of the article. For the year, I'd rather have it in both (they could be different, in theory), but the publisher really belongs to the journal.
